microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1245,7 +1284,23 @@
             }
         );
 
+        // First, hide all of the site-specific content
```

Comment:
This adds a framework for hiding/displaying certain content that should only be displayed for specific sample sites/datasets.